compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -57,10 +61,10 @@ versionDev :: Version
 versionDev = V1 PointDev
 
 supportedOutputVersions :: [Version]
-supportedOutputVersions = [version1_14, versionDev]
+supportedOutputVersions = [version1_14, version1_15, versionDev]
 
 supportedInputVersions :: [Version]
-supportedInputVersions = [version1_8, version1_11, version1_12, version1_13] ++ supportedOutputVersions
+supportedInputVersions = [version1_8, version1_11, version1_12, version1_13, version1_14] ++ supportedOutputVersions
```

Comment:
`supportedOutputVersions` already adds `version1_14`, so this should be the same

```suggestion
supportedInputVersions = [version1_8, version1_11, version1_12, version1_13] ++ supportedOutputVersions
```